### DISCUSSION TITLE
Unable to use a different redirect URL: the URL hostname of the configured OIDCRedirectURI does not match the URL hostname of the URL being accessed

### DISCUSSION THREAD

**Discussion by tpimh** [OpenIDC/mod_auth_openidc#883] [Q&A]
I'm trying to reimplement [this example](https://github.com/signicat/OIDC-MLE/blob/master/py-ftn-example/ftn-mle-example.py) with [mod_auth_openidc](https://github.com/zmartzone/mod_auth_openidc). The custom redirect URL doesn't work as intended:
```
OIDCRedirectURI https://labs.signicat.com/redirect
OIDCRedirectURLsAllowed ^https://labs\.signicat\.com/redirect$
```
This produces the following error: `oidc_authenticate_user: the URL hostname (labs.signicat.com) of the configured OIDCRedirectURI does not match the URL hostname of the URL being accessed (myhostname): the "state" and "session" cookies will not be shared between the two!`

Is it somehow possible to redirect outside the current server?

**Comment by zandbelt** [comment]
it seems you're typing "myhostname" in the browser URL bar, but you'll need to use "labs.signicat.com"

**Reply by tpimh** [reply]
I can set up local DNS for my local webserver to be accessed at labs.signicat.com, however this way I won't be able to access the original https://labs.signicat.com/redirect (this URL would be handled by an entirely different web server). Do you suggest implementing proxying for this URL?

**Reply by zandbelt** [reply]
I'm not sure what you're trying to achieve but the redirect URL handling and the regular request handling should all end up on the same web server (cluster)
